doc/02_installing.md

```diff
@@ -22,7 +22,9 @@ In `deps.edn`, create a `test` "alias" (profile) that loads the `lambdaisland/ka
 ;; deps.edn
 {:deps { ,,, }
  :aliases
- {:test {:deps {lambdaisland/kaocha {:mvn/version "0.0-122"}}}}}
+ {:test
+  {:extra-deps {lambdaisland/kaocha {:mvn/version "0.0-122"}}
+   :main-opts  ["-m" "kaocha.runner"]}}}
```

Comment:
The recommendation is now to have a `bin/kaocha` wrapper script which includes `-m kaocha.runner`. The reasoning is that this allows you to add kaocha to the classpath without necessarily invoking, e.g. if you want to have it enabled in a REPL session to run tests through Kaocha.